src/main/scala/model/QuadTreeGeo.scala

```diff
@@ -0,0 +1,83 @@
+package model
+
+case class QuadTreeGeo(level: Int = 0,
```

Comment:
Since Quad **always** has subquad, Option seems inapropriate for child representation. something with lazy initialization would be more descen